build/configs/artik05x/artik05x_download.sh

```diff
@@ -188,7 +188,7 @@ erase()
 
 	# Download all binaries using openocd script
 	pushd ${OPENOCD_DIR_PATH} > /dev/null
-	${OPENOCD} -f ${CFG_FILE} -s ${SCRIPTS_PATH} -c "flash_erase_part ${PART_NAME};	exit" || exit 1
+	${OPENOCD} -f ${CFG_FILE} -s ${SCRIPTS_PATH} -c "flash_protect off; flash_erase_part ${PART_NAME}; exit" || exit 1
```

Comment:
Do you test this with partial region like os?
When we erase os region, new command is not necessary. It needs only for bootloader.
Doesn't it cause any error?

Reply:
NO, it wont cause any error. Just that flash protection would be turned off.

Reply:
With all option, you're right. But with others, you're wrong.
When I erase userfs for example, after executing it, flash will be protected off even I didn't tough bootloader.
It's definitely wrong.
`Protect off` should be executed when we treat bootloader regions.

Reply:
Will rework soon.